client/ayon_hiero/plugins/create/create_shot_clip.py

```diff
@@ -1,247 +1,475 @@
+                tooltip="Select driving track name which should "
+                        "be mastering all others",
+                items=gui_tracks or ["<nothing to select>"],
+            ),
+
+            # publishSettings
+            UILabelDef(
+                label=header_label("Publish Settings")
+            ),
+            EnumDef(
+                "clip_variant",
+                label="Product Variant",
+                tooltip="Chose variant which will be then used for "
+                        "product name, if <track_name> "
+                        "is selected, name of track layer will be used",
+                items=['<track_name>', 'main', 'bg', 'fg', 'bg', 'animatic'],
+            ),
+            EnumDef(
+                "productType",
+                label="Product Type",
+                tooltip="How the product will be used",
+                items=['plate', 'take'],
+            ),
+            EnumDef(
+                "reviewTrack",
+                label="Use Review Track",
+                tooltip="Generate preview videos on fly, if "
+                        "'< none >' is defined nothing will be generated.",
+                items=['< none >'] + gui_tracks,
+            ),
+            BoolDef(
+                "export_audio",
+                label="Include audio",
+                tooltip="Process subsets with corresponding audio",
+                default=False,
+            ),
+            BoolDef(
+                "sourceResolution",
+                label="Source resolution",
+                tooltip="Is resloution taken from timeline or source?",
+                default=False,
+            ),
+
+            # shotAttr
+            UILabelDef(
+                label=header_label("Shot Attributes"),
+            ),
+            NumberDef(
+                "workfileFrameStart",
+                label="Workfiles Start Frame",
+                tooltip="Set workfile starting frame number",
+                default=presets.get("workfileFrameStart", 1001),
+            ),
+            NumberDef(
+                "handleStart",
+                label="Handle start (head)",
+                tooltip="Handle at start of clip",
+                default=presets.get("handleStart", 0),
+            ),
+            NumberDef(
+                "handleEnd",
+                label="Handle end (tail)",
+                tooltip="Handle at end of clip",
+                default=presets.get("handleEnd", 0),
+            ),
+        ]
+
+    def create(self, subset_name, instance_data, pre_create_data):
+        super(CreateShotClip, self).create(subset_name,
+                                           instance_data,
+                                           pre_create_data)
 
         if len(self.selected) < 1:
             return
 
-        if not widget.result:
-            print("Operation aborted")
-            return
+        self.log.info(self.selected)
+        self.log.debug(f"Selected: {self.selected}")
```

Comment:
Just for dev?

Reply:
That's adjusted :)

Reply:
I'm still seeing the log statement in the changes?